tests/eas/preliminary.py

```diff
@@ -250,9 +250,13 @@ def test_share_cap_states(self):
         """
         Check that some domain exists with SD_SHARE_CAP_STATES set
 
-        EAS silently does nothing if this flag is not set at any level (see
-        use of sd_scs percpu variable in scheduler code).
+        On old kernels (4.14 and before), EAS silently does nothing if this
+        flag is not set at any level (see use of sd_scs percpu variable in
+        scheduler code).
         """
+        if self.target.kernel_version.parts > (4, 14):
+            raise SkipTest('SD_SHARE_CAP_STATES not present on this kernel')
```

Comment:
The change is fine by me, but I'd like a more explicit message, i.e.: "Target kernel version greater than v4.14, SD_SHARE_CAP_STATES not required"